### DISCUSSION TITLE
Upload msixbundle Installers on GitHub Page

### DISCUSSION THREAD

**Discussion by Bamajon1974** [character-map-uwp/Character-Map-UWP#322] [Ideas]
Can the Character Map UWP *.msixbundle installers but uploaded on the GitHub page to avoid the possible telemetry or adware bundling issues from downloading installers directly from the MS Store according to the following links below? 

Thanks!

https://www.ghacks.net/2024/04/22/microsoft-store-installs-quicker-now-and-with-extra-telemetry/
https://www.neowin.net/news/microsoft-now-lets-you-download-app-executables-directly-from-the-microsoft-store-website/

**Comment by JohnnyWestlake** [comment]
Installing directly from the Windows Store app will not use these wrapper installers, this only occurs if you use the website (to work around security issues Microsoft imposed on themselves...)

You can otherwise get the direct install packages for any UWP store app by pasting it's web store URL here https://store.rg-adguard.net/

**Comment by Bamajon1974** [comment]
I completely forgot about the store.rg-adguard website. That solution works for me! Thank you very much!
